examples/adr_tools.php

```diff
@@ -15,10 +15,10 @@
 use StructurizrPHP\StructurizrPHP\Client\Credentials;
 use StructurizrPHP\StructurizrPHP\Client\Infrastructure\Http\SymfonyRequestFactory;
 use StructurizrPHP\StructurizrPHP\Client\UrlMap;
+use StructurizrPHP\StructurizrPHP\Core\Documentation\AdrToolsImporter;
```

Comment:
`StructurizrPHP\AdrTools\` please move here - this AdrToolsImporter does not belongs to core, it's totally different module in java.
![image](https://user-images.githubusercontent.com/1921950/68805680-97b5ed00-0664-11ea-97e4-7fca6db38ea9.png)